tools/distrib/pylint_code.sh

```diff
@@ -33,7 +33,7 @@ set -ex
 # change to root directory
 cd $(dirname $0)/../..
 
-DIRS=src/python/grpcio/grpc
+DIRS="src/python/grpcio/grpc src/python/grpcio_reflection/grpc_reflection src/python/grpcio_health_checking/grpc_health"
```

Comment:
Is this the "right" way to do a sequence of strings in `sh`? Since it's now a list of things that will grow at least a little bit more, can it be changed to be a one-element-per-line style "constant sequence of strings"?

However this is done it should pass `shellcheck`. Does what's currently drafted pass `shellcheck`?